crm_request/models/request.py

```diff
@@ -19,12 +18,14 @@ class CrmClaim(models.Model):
 
     date = fields.Datetime(string='Date', readonly=True, index=False)
     name = fields.Char(compute='_compute_name', store=True)
-    subject = fields.Char(required=True)
+    subject = fields.Char()
     alias_id = fields.Many2one(
         'mail.alias', 'Alias',
         help="The destination email address that the contacts used.")
     code = fields.Char(string='Number')
-    claim_type = fields.Many2one(string='Type')
+    claim_category = fields.Many2one(
+        "crm.claim.category", string='Category',
+        default='self.env.ref("crm_request.stage_undefined")')
```

Comment:
The default expects a string. If you want to interpret it you should use lambda function:
```suggestion
        default=lambda self: self.env.ref("crm_request.stage_undefined"))
```

Reply:
I corrected it using a compute, before seeing your message. Is it ok ?